azure/durable_functions/orchestrator.py

```diff
@@ -108,8 +111,38 @@ def _generate_next(self, partial_result):
             gen_result = self.generator.send(partial_result.result)
         else:
             gen_result = self.generator.send(None)
+
+        gen_result = self._handle_continue_as_new(gen_result)
```

Comment:
It's not obvious to me why we don't set add_to_actions to True here but we do in different places?